### ISSUE TITLE
App Window not displayed in Windows 10

### ISSUE THREAD

**Issue by henryavila** [spatie/ray#429]
**Before creating a new bug report**
Please check if there isn't a similar issue on [the issue tracker](https://github.com/spatie/ray/issues) or in [the discussions](https://github.com/spatie/ray/discussions).

If you are having troubles connecting your project to Ray, please check [our docs](https://spatie.be/products/ray) first, especially the configuration and environment specific configuration sections.

**Describe the bug**
After some use, when I open the app, the app windows is not displayed. The tray icon appears, the icon on toolbar are displayed, but no app window is displayed. I tried to open the config window and nothing.
I uninstalled the app, rebooted the computed, installed again and nothing different.
Simply cant use the app.
Sometimes, it back to work. When I open the app I never know if it will work or not

**Versions**
Ray version (you can see this in "About Ray"): 1.14.7

You can use `composer show` to get the version numbers of:
- spatie/ray package version: It's not a connection problem
- spatie/laravel-ray package version (if applicable): It's not a connection problem

PHP version:
Laravel version (if applicable): It's not a connection problem

**To Reproduce**
1. Open the app

**Expected behavior**
The app window is displayed

**Desktop (please complete the following information):**
 - OS: [e.g. macOS] MS Windows 10 PRo
 - Version [e.g. 22] 20H2 - 
 
 **Additional context**
The PC is always turned ON. IN the and of the day, I don't turn it off, just log out. In some times, the Ray app stay opened all the night. Perhaps, its linked with the issue, somehow

**Comment by noge075** [comment]
@henryavila 
I had the same issue after changing monitors. In my case I have two monitors in my home setup and three monitors in my office setup. I use Dell docking station on both places, When I came home and I used the app with "Keep this window floating" option on my third screen, I can't see Ray on on my tray. 

For me always this is the solution:
https://www.njevity.com/2014/02/18/a-window-is-not-visible-how-to-move-off-screen-windows-back-into-view/
